market/portal/meta.json

```diff
@@ -20,7 +20,7 @@
 			"archivedArtifacts": [
```

Comment:
guys; you invented this new 'archivedArtifacts' feature without asking for review,
therefore I can't tell how it works or whether it is a good idea to go there :sweat_smile: 
I'd expect you to complete the feature, so the market environment can make use of it.
see https://github.com/axonivy-market/market/pull/374